README.md

```diff
@@ -45,42 +45,42 @@ Lets walk through common flow for http request:
     actionPipe.send(new SampleHttpAction());
     // or actionPipe.createObservable(new SampleHttpAction()).subscribe(...) 
     ```
-2. Request is forwarded to `Janet` instance upon `send` or `subscribe` call;
-3. `Janet` finds suitable service to execute action and routes to it;
-4. `ActionService` knows how to deal with action and sets up `progress`/`success`/`fail` statuses;
-5. Resulting action is brought back to `Janet` instance;
-6. `Janet` routes resulting action with status to dedicated pipes;
-7. Dedicated `ActionPipe` notifies all it's observers of action wrapped with current status;
+2. The request is forwarded to the `Janet` instance upon the `send` or `subscribe` call;
+3. `Janet` finds a suitable service to execute an action and routes to it;
+4. `ActionService` knows how to deal with the action and sets up the `progress`/`success`/`fail` statuses;
+5. The resulting action is brought back to the `Janet` instance;
+6. `Janet` routes the resulting action with a status to the dedicated pipes;
+7. The dedicated `ActionPipe` notifies all its observers of the action wrapped with a current status;
 
-So `Janet` instance itself stands for routing, delegating diff job to other components:
+So the `Janet` instance itself stands for routing and delegating the diff job to other components:
 
-* `ActionPipe` – action operator, the only way to send action and receive result;
-* `ActionService` – knows how to deal with action;
-* `ActionServiceWrapper` – decorator to put additional logic for underlying service;
+* `ActionPipe` – an action operator, the only way to send an action and receive a result;
+* `ActionService` – knows how to deal with an action;
+* `ActionServiceWrapper` – a decorator to add additional logic to the underlying service;
 
 ### Available services
-Janet abilities depends on services.
-Currently there are:
+The Janet abilities depend on services.

```

Comment:
I believe `Janet` acts here on it's own, like a name or title, so we can omit _the_ article. I've checked through some popular libs, and seems to be valid. E.g. http://square.github.io/retrofit/ -> `the Retrofit class` but `Retrofit turns your HTTP API...`

Reply:
English is damn tricky sometimes ;-))) If you mention names without other words specifying them like here: Retrofit turns your HTTP - no article is needed (no word going after Retrofit). But if you'd like to specify it somehow like here: the Retrofit class - there's an article simply because there's a word 'class' specifying the name 'Retrofit'.